### PR TITLE
added support in modular-tokenizer to pure text meta instructions directive for controlling max tokens length locally per section

### PR DESCRIPTION
**Pull request by YoelShoshan**
For example:

**<@TOKENIZER-TYPE=AA@MAX-LEN=121>**EIVMTQSPGTLSLSPGERATLSCRASQSISSSYLAWFQQKPGQAPRLLIYGASSRAPGIPDRFSGSGSGTDFTLTISRLEPEDFAVYYCQQYGDSPLTFGGGTKVDIKRTVAAPSVFIFPPSDEQLKSGTASVVCLLNNFYPREAKVQWKVDNALQSGNSQESVTEQDSKDSTYSLSSTLTLSKADYEKHKVYACEVTHQGLSSPVTKSFNRGEC<MOLECULAR_ENTITY><MOLECULAR_ENTITY_ANTIBODY_LIGHT_CHAIN>EVQLQQWGAGLLKPSETLSLTCAVYGES**<@TOKENIZER-TYPE=AA>**<MOLECULAR_ENTITY><MOLECULAR_ENTITY_ANTIBODY_LIGHT_CHAIN>EVQLQQWGAGL**<@TOKENIZER-TYPE=AA@MAX-LEN=234>**<MOLECULAR_ENTITY><MOLECULAR_ENTITY_ANTIBODY_LIGHT_CHAIN>EVQLQQWGAGL

### CODE REVIEW

File: fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py [hunk 1]
```diff
@@ -1007,12 +1011,25 @@ def encode(
             len(hints_and_subseq) > 0 and len(hints_and_subseq) % 2 == 0
         ), f"Error: expecting leading modular tokenizer hints followed by a sequence to tokenize, got {sequence}"
         # arrange as a list of TypedInput - each one will include the type and the following sequence
-        encode_list_format = [
-            TypedInput(tokenizer_type, subseq, None)
-            for tokenizer_type, subseq in zip(
-                hints_and_subseq[::2], hints_and_subseq[1::2]
-            )
-        ]
+        encode_list_format = []
```
Comment (mosheraboh) [inline fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py:1014]: fix comment in line 1006

**Review by mosheraboh — APPROVED** [review]
Nice!
One minor inline.

**Review by floccinauc — COMMENTED** [review]
Looks good

File: fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py (hunk 1 above)
Comment (YoelShoshan) [reply fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py:1014]: thanks! done

**Pull request merged**